src/templates/rollup/index.js

```diff
@@ -0,0 +1,316 @@
+import _ from 'lodash';
+
+import { getFormattedCode } from '../../utils/formatter';
+
+export const getReadMeFile = (name, features) => {
+  const isReact = _.includes(features, 'React');
+  return getFormattedCode(
+    `
+## ${name}
+
+## Building and running on localhost
+
+First install dependencies:
+
+\`\`\`sh
+npm install
+\`\`\`
+
+${
+  isReact
+    ? `
+
+To create a development watch the react app in browser
+
+\`\`\`sh
+npm start
+\`\`\`
+`
+    : `
+To create a development in watch mode:
+
+\`\`\`sh
+npm run watch
+\`\`\`
+`
+}
+
+
+
+To create a production build:
+
+\`\`\`sh
+npm run build
+\`\`\`
+
+
+## Credits
+
+Made with [createapp.dev](https://createapp.dev/)
+
+  `,
+    'md'
+  );
+};
+
+export const getIndex = configItems => {
+  const isBabel = _.includes(configItems, 'Babel');
+
+  return getFormattedCode(
```

Comment:
Could this call, in theory, be slow? If it makes external API calls or something? If it is then we need to consider doing caching.

Reply:
No, it's just calling a js function which is inside utils folder. So it won't cause any delay.

Reply:
These apis are removed in latest commit, somehow the this commit is making some issue in my system. So I corrected the indentation manually. I will try to analyse more once this PR is merged.